common_test.go

```diff
@@ -4,22 +4,54 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"os/exec"
 	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
 )
 
-type SimpleLinguistTestSuite struct {
+const lingustURL = "https://github.com/github/linguist.git"
+
+type EnryTestSuite struct {
 	suite.Suite
+	tmpLinguist string
+}
+
+func TestEnryTestSuite(t *testing.T) {
+	suite.Run(t, new(EnryTestSuite))
+}
+
+func (e *EnryTestSuite) SetupSuite() {
+	tmpLinguist, err := ioutil.TempDir("", "linguist-")
+	assert.NoError(e.T(), err)
+	e.tmpLinguist = tmpLinguist
+
+	cmd := exec.Command("git", "clone", lingustURL, tmpLinguist)
```

Comment:
Actually it is like an integration test.

To generate the files, we have already cloned linguist in the project's directory. Maybe it's better perform a checkout over this repo, do the test and then come back to the master of this repo.

Should I do something like this?